### PR TITLE
A js Todo-App with using webpack and node js

### PR DESCRIPTION
**Pull request by MahmoudBakr23**
I've built three js files
- index.js
- project.js
- todo.js
the project and index are exported to the index.js and the whole src folder is bundled to dist/main.js
- the Project class is for creating and displaying a project
- the Todo class is for creating and displaying a todo in each project specifically
NOTE>>"I haven't done the task expansion as mentioned in the Odin project tasks for styling matter (my opinion) if still required, I'll do it right away, Thanks for understanding!"

### CODE REVIEW

File: src/project.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+const projects = [];
```
Comment (christianotieno) [inline src/project.js:1]: ## ADD a default project
- [ ] According to the project's requirements, there should exist a `default` project on the initial page load. Currently, we don't have this in our project, let's add it.
#### ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/projects_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)).

File: styles/style.css [hunk 2]
```diff
@@ -1 +1,58 @@
-/* No CSS *//*# sourceMappingURL=style.css.map */
\ No newline at end of file
+#list {
```
Comment (christianotieno) [inline styles/style.css:1]: ###  \[OPTIONAL\] Stretch Requirements Checklist & Common issues
- [ ] Do you mind improving the UI in terms of contrast? 😟 Currently, it is a bit of a struggle locating the input sections of the form.

**Review by christianotieno — CHANGES_REQUESTED** [review]
### Changes Required ♻️ 
Hello @MahmoudBakr23  & @anewman15  👋 

Good job with the todo project. We have with us a well organised, modularised and professional project. Good job using es6 classes. We can, however, improve on it by replacing most of our methods with arrow-based functions. 
##
###  \[OPTIONAL\] Stretch Requirements Checklist & Common issues

- [ ] Do mind adding this optional additional script to your webpack config for an easier startup of the project? Thanks 

`"start": "webpack-dev-server --mode development --open" // optional`
##

Find below comments on areas we need to improve on before we continue. 👇 

### Happy coding 💻  

#### Christian Otieno (TSE)

**Comment by MahmoudBakr23** [comment]
Thank you for reviewing, I'll note down the optional requests

**Review by rOluochKe — APPROVED** [review]
Hello @MahmoudBakr23 
### You got it right, well done. The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**